behave/docker-compose.m1.yml

```diff
@@ -0,0 +1,13 @@
+version: '3.4'
+
+services:
+  seleniumchrome:
+    image: seleniarm/standalone-chromium:4.0.0-beta-1-20210215
+
+  clafrontend:
+    platform: linux/arm64
+    build: 'https://github.com/ministryofjustice/cla_frontend.git#create-unittests-block'
+
+  clapublic:
```

Comment:
Because `clabackend` actually builds and runs fine on an arm64 machine without the need of specifying the platform. It doesn't use anything that conflicts or is outdated, so therefor we don't need to override the docker-compose.yml `clabackend` config.

Reply:
I can add this to the Read.me to outline this if that's better?

Reply:
yes in readme please.